Makefile

```
@@ -0,0 +1,15 @@
+init: migrate
+	go build main.go
+
+migrate:
+	cp .env.example .env
+	cd db
+	goose sqlite3 ./big_bang_theory_quotes.db up
```

not everyone may have the db file inside the "db" directory, can we define variables instead?

```suggestion
	// DB_MIGRATIONS_DIR = $(PWD)/db/migrations
	// SQLITE_DSN comes from the env file
	goose -dir  $(DB_MIGRATIONS_DIR) sqlite3 $(SQLITE_DSN) up
```

---

Woah, never thought of that `include` keyword to install the env on Linux.  

Thank you.